sdk/state/integration/state_test.go

```diff
@@ -41,8 +41,13 @@ func TestMain(m *testing.M) {
 }
 
 func TestOpenUpdatesUncoordinatedClose(t *testing.T) {
-	initiator, responder := initAccounts(t)
-	initiatorChannel, responderChannel := initChannels(t, initiator, responder)
+	asset := txnbuild.NativeAsset{}
```

Comment:
these variables are needed because of the signature change of `initAccounts`. It's not changing anything though since we're still just using the native asset